lib/utils/embeddings/embed.ts

```diff
@@ -42,16 +43,17 @@ export default async function addRepo(
 			process.env.GITHUB_ACCESS_TOKEN || ''
 		)
 
-		const docs = repo.map(doc => {
+		const docs = Promise.all(repo.map(async doc => {
```

Comment:
`.map(async () => { await someFunc() }` might not actually wait 

I believe the only JS for loop that respects async-await is `for (const doc or repo) { await someFunc() }`

Reply:
.map(async () => { await someFunc() } -> Why does it not?